### DISCUSSION TITLE
thanhle.blog/thuat-toan-frontend-tim-node-chua-content-chinh

### DISCUSSION THREAD

**Discussion by giscus** [thanhlmm/blog#69] [Ideas]
# thanhle.blog/thuat-toan-frontend-tim-node-chua-content-chinh

Liệu code frontend chỉ cần đúng theo design là đủ? Thuật toán đồ là dbrr?

https://thanhle.blog/blog/thuat-toan-frontend-tim-node-chua-content-chinh

**Comment by nguyentrungson97** [comment]
hóng bài anh chửi Next 13 =))

**Reply by chungnq253** [reply]
+1 hóng bài chửi Next 13 :v
